client/network/network.js

```diff
@@ -14,14 +14,11 @@ limitations under the License.
 ==============================================================================*/
 
 import io from '/lib/lame_es6/socket.io-client.js';
-import Debug from '/lib/lame_es6/debug.js';
 import * as info from '/client/util/info.js';
 import * as time from '/client/util/time.js';
-
-const debug = Debug('wall:network');
+import {installMessageHandler, wrapMessageSocket, cleanupMessageHandlers} from '../../lib/socket_wrapper.js';
```

Comment:
These names are too generic.
They either want a parent like "modulenet.js" or better names like installModuleOverlayHandler

Reply:
Went with the rename.